tests/hdkeytests/hdkey_mnemonic_test.c

```diff
@@ -20,18 +20,18 @@ static void test_build_wordlist(void)
     for (int i = 0; i < 9; i++) {
         char wmnemonic[256];
         const char *lang = languagelists[i];
-        const char *mnemonic = Mnemonic_Generate(lang);
```

Comment:
why change to  use "nmnemonic" instead of origin one, where "mnemonic" is the one with right word.